pepper-apis/src/main/resources/db-changes/schema/DDP-3046-create-event-service.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="rnordin" id="20191120-drop-activity-condition">
```

Comment:
I see rows in dev for this table.  Where has the data/functionality gone?  Is there migration left to do here, or do we no longer need this capability?

Reply:
It was a very rudimentary functionality which wasn't used by the time angio went live:

Activities don't need to know under what circumstances they should create themselves. Rather EventConfigurations will determine the circumstances when activities will be created.